.gitignore

```diff
@@ -4,3 +4,6 @@
 /production
 /puzzles
 /testing
+
+/bazel-*
+.clwb
```

Comment:
What is .clwb? Also, please add a newline at EOF.

Reply:
Added a new line at EOF. `.clwb` is a project directory created by Intellij (CLion) if you use the Bazel plugin. It contains metadata about the project that bridges Bazel and IntelliJ project models.

Reply:
Oh, I see... We appreciate the fix, but I'm going to reject it because: 1) We actually aren't accepting external contributions yet (as mentioned in the CONTRIBUTING.md file). 2) Regardless of that, it wouldn't be a productive approach to try to accommodate all possible tools that may create some files or directories in the checkout: instead, we recommend that every user sets up [a global .gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore) on their system.